openlibrary/core/vendors.py

```diff
@@ -120,10 +120,18 @@ def get_products(self, asins, serialize=False, marketplace='www.amazon.com',
                                       resources=_resources,
                                       **kwargs)
         except ApiException:
-            logger.error("Amazon fetch failed for: %s" % ', '.join(item_ids),
-                         exc_info=True)
+            logger.exception("Amazon fetch failed for: %s" % ', '.join(item_ids))
             return None
-        response = self.api.get_items(request)
+        response = None
```

Comment:
I think this line is unnecessary. `response` is either not used, or needs to be an object with properties.